src/help.cpp

```diff
@@ -99,8 +99,8 @@ void help::draw_menu( const catacurses::window &win )
 Please press one of the following for help on that topic:\n\
 Press ESC to return to the game." ) ) + 1;
 
-    size_t half_size = help_texts.size() / 2;
-    int second_column = getmaxx( win ) / 2;
+    size_t half_size = help_texts.size() / 2 + 1;
```

Comment:
Not sure what this is supposed to be calculating exactly, but (depending on what you're trying to achieve here) perhaps rather than adding 1 the correct solution is to use `divide_round_up`?

Reply:
This calculates the length of the columns for a proper division - without it the second column gets messed up. `divide_round_up` doesn't have an overload for `size_t` so I won't be able to use it in this line.

Reply:
To be clear, would `divide_round_up` have been a sensible solution if a suitable overload existed?